Install dep bug

fitz123 opened the pull request:
Got this:

```
TASK: [keepalived | install dependencies] ************************************* 
failed: [192.168.42.51] => (item=keepalived) => {"failed": true, "item": "keepalived"}
stderr: E: There are problems and -y was used without --force-yes

stdout: Reading package lists...
Building dependency tree...
Reading state information...
The following extra packages will be installed:
  iproute ipvsadm libperl5.18 libsensors4 libsnmp-base libsnmp30
Suggested packages:
  heartbeat ldirectord lm-sensors snmp-mibs-downloader
The following NEW packages will be installed:
  iproute ipvsadm keepalived libperl5.18 libsensors4 libsnmp-base libsnmp30
0 upgraded, 7 newly installed, 0 to remove and 46 not upgraded.
Need to get 1221 kB of archives.
After this operation, 4728 kB of additional disk space will be used.
WARNING: The following packages cannot be authenticated!
  libsensors4 libperl5.18 ipvsadm keepalived

msg: '/usr/bin/apt-get -y -o "Dpkg::Options::=--force-confdef" -o "Dpkg::Options::=--force-confold"   install 'keepalived'' failed: E: There are problems and -y was used without --force-yes
```

tersmitten commented:
On what platform /version are you? `force` shouldn't be needed. Is it possible that you're missing `keyring` [packages](http://ubuntuforums.org/showthread.php?p=7001019#7001019)?

tersmitten commented:
@fitz123: Did you figure it out?

fitz123 commented:
@tersmitten, yes, i fixed that. The reason was:
`deb mirror://mirrors.ubuntu.com/mirrors.txt`
as all my repos. Its super cool specially for a country where you have shitty international connection =) But works not 100%. So eventually I've replaced all repos by normal ones and it never happened again

And yes, there's nothing about your role specifically
